package NAlice.NWonderSdk;

option go_package = "a.yandex-team.ru/alice/wonderlogs/sdk/app_converter/protos";

enum EAppType {
    AT_UNDEFINED = 0;
    AT_BROWSER_ALPHA = 1;
    AT_BROWSER_BETA = 2;
    AT_BROWSER_PROD = 3;
    AT_STROKA = 4;
    AT_YABRO_BETA = 5;
    AT_YABRO_PROD = 6;
    AT_LAUNCHER = 7;
    AT_SMALL_SMART_SPEAKER = 8;
    AT_NAVIGATOR = 9;
    AT_SEARCH_APP_BETA = 10;
    AT_SEARCH_APP_PROD = 11;
    AT_ELARIWATCH = 12;
    AT_AUTO = 13;
    AT_QUASAR = 14;
    AT_OTHER = 15;
}
